FSW/util/memUtil.cpp

```diff
@@ -0,0 +1,60 @@
+/* memUtil.cpp handles NS2 payload memory allocation and assignment 
+ * Usage:
+ *  module functionality
+ *  function definitions
+ *  put function declarations in memUtil.hpp
+ * 
+ * Modules encompassed:
+ *  Data Processing
+ *  Science Memory Handling
+ *
+ * Additional files needed for compilation:
+ *  none
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+#include "../headers/memUtil.hpp"
+
+/* - - - - - - Module Driver Functions - - - - - - */
+
+/* - - - - - - dataProcessing - - - - - - *
+ * Usage:
+ *  Reads incoming data from ADC, measures against baseline,
+ *  appends time and pointing data, sends to memory.
+ * 
+ * Inputs:
+ *  none
+ *  
+ * Outputs:
+ *  None
+ */
+void dataProcessing()
+{
+    std::cout << "Data Processing Module: Test Print" << std::endl;
+}
+
+
+/* - - - - - - scienceMemoryHandling - - - - - - *
```

Comment:
Big fan of these function headers, good for code review and important to be able to quickly glance and what functions in main do

Reply:
I tend to put them inside the function definition but it doesn't really matter as long as it is consistent